src/renderer/components/RandomizerTile/LuckyBeggarList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <ul class="wrapper" v-if="luckyBeggarList.length > 0">
+    <li class="field" v-for="(item, index) in getLastRecords(luckyBeggarList)" v-bind:key="index">
```

Comment:
See my comment below, but this line can look like this:
`v-for="(item, index) in getLastRecords" :key="index"`

Extras: maybe there is a better property than index for `key`?

Reply:
You're right - let's keep things simple.